bin/ufcc

```diff
@@ -63,6 +63,91 @@ with warnings.catch_warnings(record=True) as w:
         print(mass_warn)
         warn2report.append(mass_warn)
 
+# For interactive selection of the groups for the contacts calculation
+if args.i_bool:
+    groups_dict = {
```

Comment:
You should consider putting the code into a separate module so it can have its own API and be called outside of the command line.

Reply:
Done! I isolated the code for the interactive selection in a function in the `ufcc.interactive_sel` module.
You can use it simply by calling this function as in:
```python
from ufcc import UFCC
from ufcc.interactive_sel import interactive_selection
from ufcc.sampledata import GIRK

target_system = UFCC(GIRK.coordinates, GIRK.trajectory)
target_system = interactive_selection(target_system)
```